app/providers/megakino/sitemap.py

```
@@ -0,0 +1,190 @@
+        return None
+    _, slug = tail.split("-", 1)
+    slug = slug.strip("/ ")
+    if not slug:
+        return None
+    return slug, kind
+
+
+def parse_sitemap_xml(xml_text: str) -> List[MegakinoIndexEntry]:
+    """Parse a megakino sitemap XML string into index entries."""
+    entries: List[MegakinoIndexEntry] = []
+    root = ET.fromstring(xml_text)
```

Ruff (0.14.10)

Using `xml` to parse untrusted data is known to be vulnerable to XML attacks; use `defusedxml` equivalents

(S314)